programs/protocol-contracts-solana/src/lib.rs

```diff
@@ -44,15 +46,30 @@ pub mod gateway {
         chain_id: u64,
     ) -> Result<()> {
         let initialized_pda = &mut ctx.accounts.pda;
+
         initialized_pda.nonce = 0;
         initialized_pda.tss_address = tss_address;
         initialized_pda.authority = ctx.accounts.signer.key();
         initialized_pda.chain_id = chain_id;
         initialized_pda.deposit_paused = false;
+        initialized_pda.deposit_fee = 2_000_000; // 0.002 SOL
```

Comment:
@skosito: if we add more stuff to the pda account, we need a way to "migrate"--as the contract is already deployed on testnet.  The contract cannot be initialized twice. 

Alternatively, we scratch it and deploy to a new contract address on testnet.

Reply:
We could, but to be correct we also need to migrate the SOL and nonce from old program to new program, essentially migrate the old PDA to new PDA...

Reply:
Can we maybe just hardcoded it and we upgrade every time the value must be updated (I would guess we're not likely to update the value often). Update the value as a variable already require admin privilege anyway